### ISSUE TITLE
Code splitting: React.lazy() example is missing a Suspense wrapper

### ISSUE THREAD

**Issue by manuelbieh** [reactjs/reactjs.org#1810]
I'm not quite sure if this is intentional to only demonstrate how `React.lazy()` is used, if the API how lazy() works was changed after that part of the docs have been written already, if I am using it incorrectly or if it is actually wrong but in the example but in the code splitting guide you're using a component wrapped in a `React.lazy()` and do not wrap it in a `<Suspense>`:

https://reactjs.org/docs/code-splitting.html#reactlazy

When I do that I get an error message:

> A React component suspended while rendering, but no fallback UI was specified.

 [Source](https://codesandbox.io/s/zknlo3z224)

Two paragraphs later Suspense is introduced and then a working example is shown where the lazy-loaded component is correctly wrapped in `<Suspense>` but I don't know if it is a good idea to have a non-functioning example in the documentation in the first place as it might be misleading.

**Comment by sairion** [comment]
I am also confused by this message.

**Comment by nico-bellante** [comment]
I also used this example just to get the same error. Luckily, googling it took me here :)

**Comment by helloguille** [comment]
Were you able to use React.lazy without Suspense?

**Comment by nico-bellante** [comment]
Nope, I got the error above and wrapped it in a Suspense to fix it

**Comment by allenchuang** [comment]
Getting the same issue as well..

**Comment by Tanzeel550** [comment]
> Nope, I got the error above and wrapped it in a Suspense to fix it

Where did Suspense came from?

**Comment by helloguille** [comment]
I checked the docs and I think the issue is now fixed.
